Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -705,14 +703,72 @@ extension SPNoteEditorViewController: InterlinkProcessorDelegate {
 }
 
 
+// MARK: - Tags
+//
+extension SPNoteEditorViewController {
+    @objc
+    func configureTagListViewController() {
+        tagListViewController = NoteEditorTagListViewController(note: note)
+        addChild(tagListViewController)
+        let tagView = tagListViewController.view!
+
+        tagView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(tagView)
+
+        NSLayoutConstraint.activate([
+            tagView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tagView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        tagListBottomConstraint = tagView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        tagListBottomConstraint.isActive = true
+
+        tagListViewController.didMove(toParent: self)
```

Comment:
Not sure if it's 100% needed, but missing call to `willMove(toParent:)`

Reply:
When you add a child view controller, `willMove(toParent:)` is not needed. 
Ref: https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html